### DISCUSSION TITLE
v1.8.4

### DISCUSSION THREAD

**Discussion by yujincheng08** [LSPosed/LSPosed#2115] [New Release]
### Changelog
- Tested support for Android 13
- Fix module preference on non-primary users
- Fix log saving
- Apply the SELinux label of the entire WebView cache directory
- Refactor core
- Fix `cancelNotificationWithTag` for Android 11-
- Fix `broadcastIntentWithFeature` on Android 12+
- Manager targets SDK 33
- Replace the corruption handler of DB to avoid crashing
- Use abseil to improve performance
- Display module/releases last publish time
- Fix deoptimize static methods
- Preliminary support for Android 14

### 更新日志
- 测试 Android 13 兼容性
- 修复非主用户的模块配置
- 修复日志保存
- 整个应用 WebView 缓存目录的 SELinux 标签
- 重构核心
- 修复 Android 11- 的 `cancelNotificationWithTag`
- 修复 Android 12+ 的 `broadcastIntentWithFeature`
- 管理器目标 SDK 33
- 替换数据库的损坏句柄以避免崩溃
- 使用 abseil 提升性能
- 显示模块/发行版的发布时间
- 修复静态方法的去优化
- 初步 Android 14 支持

### Contributors

@vvb2060 @canyie @yujincheng08 @Dr-TSNG @chenxiaolong @1552980358 @TigerBeanst @dependabot @keta1 @1552980358 @fankes @Howard20181

<hr /><em>This discussion was created from the release <a href='https://github.com/LSPosed/LSPosed/releases/tag/v1.8.4'>v1.8.4</a>.</em>

**Comment by Ikkegwn** [comment]
So, this version, I don't know what it does but on an app, example is pokemon masters, i get errored and can't play, even with magisk deny list. I downgraded to 1.8.3 and it works again

**Reply by Howard20181** [reply]
Tested with no problems, please note that Magisk denylist has no hidden function.

**Comment by Ikkegwn** [comment]
Strangely enough, when i rolled back, it worked again.
I do know that pokemon masters, when it detects root, it just crashes.
Now, with 1.8.4, i get errors and can't get ingame. Might just be a specific problem and maybe it's just me.
________________________________
